torchgeo/datasets/geo.py

```diff
@@ -342,7 +346,11 @@ class RasterDataset(GeoDataset):
     #: ``start`` and ``stop`` groups.
     date_format = "%Y%m%d"
 
-    #: True if dataset contains imagery, False if dataset contains mask
+    #: True if the dataset contains source data, such as imagery. False if the dataset
+    #: contains target data, such as a mask. This is the same as Kornia. When multiple
```

Comment:
> This is the same as Kornia

It isn't clear _what_ is the same here. The "image" and "mask" keys are the same, but you don't mention those until later.

Reply:
How about this:
> True if the dataset contains source data, such as imagery. False if the dataset contains target data, such as a mask. In order to maintain consistency with Kornia, if ``is_image`` is set to ``True``, ``__getitem__`` uses ``image`` as the key for the data; otherwise, ``__getitem__`` uses ``mask`` as the key.
?

Reply:
I committed the change for your review.